public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -349,7 +361,7 @@ export class QueryEditorRow<TQuery extends DataQuery> extends PureComponent<Prop
     const DatasourceCheatsheet = datasource.components?.QueryEditorHelp;
 
     return (
-      <div aria-label={selectors.components.QueryEditorRows.rows}>
+      <div aria-label={selectors.components.QueryEditorRows.rows} onKeyDown={this.runOnKeyDown}>
```

Comment:
`aria-label`should not be used on divs. If the element has a semantic value, use a landmark element that fits it, otherwise change this to `data-testid` (Assuming it is for e2e tests)
```suggestion
      <div data-testid={selectors.components.QueryEditorRows.rows} onKeyDown={this.runOnKeyDown}>
```

`selectors.components.QueryEditorRows.rows` needs to be prefixed with `data-testid` for this to work

Reply:
Oh okay. Not sure why we use aria-label here then, i'll change it 👍

Reply:
But how do i prefix it with `data-testid`? And the e2e test specifically looks for aria-labels but then i'll have to change the tests

Reply:
Just add it to the string here https://github.com/grafana/grafana/blob/e76848acee2635fd99a07bfdc25d57106bcc08dd/packages/grafana-e2e-selectors/src/selectors/components.ts#L139